builder.py

```diff
@@ -249,6 +258,103 @@ def build_notes(self):
         write_htaccess(notes_path, settings.CANONICAL_URL + helper.thunderbird_url('releasenotes'))
         write_htaccess(beta_notes_path, settings.CANONICAL_URL + helper.thunderbird_url('releasenotes', channel="beta"))
 
+        self.build_notes_feed(feed_items)
+
+    def build_notes_feed(self, feed_items):
+        """ Builds the release notes atom.xml file. Like build_notes, this is en-US only. """
+        if len(feed_items) == 0:
+            return
+
+        # RSS follows the notes, being only en-us
+        if self.lang != 'en-US':
+            self._switch_lang('en-US')
+
+        def sort_version_by_release_date(i):
+            """ Sort using the note's release date. """
+            return i[1]['release'].get('release_date')
+
+        # Sort notes by release date
+        feed_items.sort(key=sort_version_by_release_date, reverse=True)
+
+        releases = list(filter(lambda i: 'beta' not in i[0], feed_items))
+        betas = list(filter(lambda i: 'beta' in i[0], feed_items))
+
+        # We only want to display the last 10 releases, and the last 5 betas
+        feed_items_mixed = releases[:10] + betas[:5]
+        # Sort again by release date
+        feed_items_mixed.sort(key=sort_version_by_release_date, reverse=True)
+
+        feed_template = self._env.get_template('includes/atom-feed.html')
+        content_template = self._env.get_template('includes/release-notes-feed.html')
+
+        self._env.globals.update(feedback=releasenotes.settings["feedback"], bugzilla=releasenotes.settings["bugzilla"])
+
+        fake_context = {'LANG': 'en-US'}
+        feed_context = {
+            'feed_url': os.path.join(settings.CANONICAL_URL, 'thunderbird', 'releases', 'atom.xml'),
+            'feed_icon': f'{settings.CANONICAL_URL}{settings.MEDIA_URL}/img/thunderbird/favicon-196.png',
+            'feed_logo': f'{settings.CANONICAL_URL}{settings.MEDIA_URL}/img/thunderbird/thunderbird-256.png',
+            'feed_index': f'{settings.CANONICAL_URL}/{helper.url(fake_context, 'thunderbird.releases.index')}'
```

Comment:
This line errors for me, think single quotes inside single quotes is bad.

Reply:
I'm surprised and slightly afraid it doesn't error out for me. But thanks for catching that!